index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Consul</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#"><img src="./img/consul-logo.png" alt="" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./img/Hamburger_icon.svg.png" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="home-section">
+        <div class="home-wrapper">
+          <div class="home-section-header">
+            <p class="home-greeting">"Hello! Sharing World"</p>
+            <h1 class="home-info">
+              Unlocking the potential of those who advance the world
+            </h1>
+          </div>
+          <div class="home-text">
+            <p>
+              We increase our clients' topline by optimizing their growth
+              strategies, marketing, pricing and sales.
+            </p>
+          </div>
+          <div class="home-timetable">
+            <p>We work from (MON) ~ (SUN)</p>
+          </div>
+          <div class="home-adress">
+            <p>We build a relationship that’s based on trust. Over the years they’ve proven that their delivery is fast and stable</p>
+          </div>
+
+        </div>
```

Comment:
- [ ] The subheading, `Hello sharing world` should be grouped using a header tag, from `h2` and below instead of a `p` tag. 
- [ ] Reduce the home-address' font size to about 20px to match the given design as well as page contrast improving the page's accessibility.